### DISCUSSION TITLE
How to try scanner beta?

### DISCUSSION THREAD

**Discussion by gbeyer3** [AlexandreRouma/SDRPlusPlus#916] [Q&A] [answered]
It appears that the beta scanner feature is available.  I just downloaded and am running the nightly build from 11/12/22.  Where is the scanner feature?  Is it a plugin I have to install separately?  Any guidance appreciated.  Thanks!

**Comment by gbeyer3** [comment]
Really, no one has tried the scanner?

**Reply by AlexandreRouma** [reply]
See the relevant section in the manual: https://www.sdrpp.org/manual.pdf

**Reply by AlexandreRouma** [reply, marked as answer]
(the relevant sections being "Scanner" and "Adding and removing modules")

**Reply by gbeyer3** [reply]
Had no idea there WAS a manual for SDR++, been just learning it trial and error . . . this is GREAT.  Thanks!

**Reply by jcoles56** [reply]
I didn't know there was a manual. Thanks! 
A manual is very necessary for the Scanner feature because trying to work it like a conventional scanner radio produces unexpected results. "The step size is random" and "it won't stop on a signal" were my first reactions. It seemed broken. But I'm getting used to it now. So far, I have only tried AM and FM broadcast bands. I intend to try it out in more dynamic bands like Ham radio or Aircraft soon.

**Reply by jcoles56** [reply]
Looking at my 4-Dec 2022 posting, I must have found a way back then to get the scanner to stop on stations. But today, trying it on the FM band with the latest version of SDR++, I'm back to it just stepping through the range according to the interval, never stopping. The Level control makes no difference at all. 

Are people using this feature? What settings have worked for you?

**Reply by jcoles56** [reply]
Selecting a higher sample rate on my AirspyHF+ improved scanner performance. I had it set to 192kHz. Max is 912kHz. Now the scanner actually stops on signals that exceed the Level setting.
